(docs): Fix vale warnings (#5225)
kedro-org/kedro: docs/configure/config_loader_migration.md

```

### 4. File format support

> `OmegaConfigLoader` supports `yaml` and `json` file formats. Make sure that all your configuration files use one of these formats. If you relied on other formats with `ConfigLoader`, convert them before upgrading.

Reviewer: ```suggestion
`OmegaConfigLoader` supports `yaml` and `json` file formats. Make sure that all your configuration files use one of these formats. If you relied on other formats with `ConfigLoader`, convert them to `yaml` or `json`.
```
